terraform/testcases/prometheus_mock/otconfig.tpl

```diff
@@ -5,7 +5,7 @@ receivers:
   prometheus:
     config:
       global:
-        scrape_interval: 15s
+        scrape_interval: 1s
```

Comment:
Doesn't changing the scrape interval affect performance? Why is this change necessary?

Reply:
yes it should affect performance by factor of 15. This is to match the metric load as mentioned in the performance report.

Reply:
Reverted. Will pull into separate PR as discussed.